packages/core-jest-matchers/src/transactions/types/delegate.ts

```diff
@@ -17,7 +17,8 @@ expect.extend({
     toBeDelegateType: received => {
         return {
             message: () => "Expected value to be a valid DELEGATE transaction.",
-            pass: received.type === DelegateRegistration,
+            pass: received.type === TransactionTypes.DelegateRegistration ||
+                  received.type === TransactionTypes.DelegateResignation,
```

Comment:
This matcher is supposed to check for a delegate registration, this will now cause false results for tests if you don't pay attention. All changes in this file need to be reverted.

There is already a matcher for a resignation at https://github.com/ArkEcosystem/core/blob/b58f7dcfca73a34158f7d76486bcf3e11dcfa596/packages/core-jest-matchers/src/transactions/types/delegate-resignation.ts.